src/fhirtypes/ElementDefinition.ts

```diff
@@ -791,6 +806,39 @@ export class ElementDefinition {
     if (disabledFlags.length) {
       throw new DisableFlagError(disabledFlags);
     }
+    let newStatusExtension: any = null;
+    const currentStatusExtension = this.extension?.find(
+      extension =>
+        extension.url ==
+        'http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status'
+    );
+    if (currentStatusExtension && (trialUse || normative || draft)) {
+      throw new ExistingStandardsStatusError(this.id);
```

Comment:
Hmm, that's a good point, especially in the context of the standards status of the StructureDefinition itself. If overriding an existing standards status is allowed, and nothing else about the current implementation changes, then something like this ends up as allowed:
```
* somePath TU
* somePath N
```
unless additional checks are added for "is there more than one rule on this profile that sets the standards status for this element," since once the flag rule is applied, the extension on the element has no indication of its original source.

Reply:
That's true, that would be allowed. But if we decide that something like that is valid, is it a problem if it is allowed? I think there are other cases where we allow things that are technically valid, but maybe pretty dumb. Like for example I think I could do:
```
Profile: MyProfile
Parent: Observation
* status = http://system.com#code
* basedOn 0..10
* basedOn 0..9
* basedOn 0..8
```
if I was so inclined. Which strikes me as a similar situation.